### PR TITLE
Revert add noarch

### PR DESCRIPTION
**Pull request by gabm**
This is a fix to #9 that reverts #8 

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jschueller** [comment]
you need to set build number to 2 so as the new package is taken into account

**Comment by gabm** [comment]
of course.. you're right.. done

**Comment by gabm** [comment]
I did the same here: https://github.com/conda-forge/idna-feedstock/pull/9

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,15 +12,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
```
Comment (jakirkham) [inline recipe/meta.yaml]: We should be able to save this `pip install` line.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,15 +12,13 @@ source:
...
+  number: 2
+  script: python setup.py install --single-version-externally-managed --record=record.txt
   entry_points:
     - chardetect = chardet.cli.chardetect:main
 
 requirements:
-  host:
-    - pip
+  build:
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also changing to `host` and using `pip` here should be ok.

**Comment by jakirkham** [comment]
Thanks @gabm.

There are a few things we would like to avoid reverting from those PRs, but they are less pressing. So we can follow-up on them as needed, but would be good if new reversion PRs took these into account.

**Pull request merged**